viz/app/components/VisualizationWrapper.tsx

```diff
@@ -266,12 +278,17 @@ export function VisualizationWrapper({
 
   return (
     <div className="relative group/viz">
+      <div className="flex flex-row gap-2 absolute top-2 right-2 bg-white rounded transition opacity-0 group-hover/viz:opacity-100 z-50">
```

Comment:
@overmode I understand your maintained here the design of the button as it was, and very well aware that we don't have access to sparkle here, but don't hesitate when you see something to fix it.

(operating principle: "We avoid the bystander effect: “see it, say it, solve it”.", see https://docs.google.com/document/d/1YIRfpUvh8hHzt-TnvAn1qHnz_F65b-OC8o_1b1kg8IU/edit?tab=t.0)

I think it's an easy fix to align the design here more with sparkle. In particular the close button of the Modal component is likely easy to reproduce with simple tailwind here :+1: 

Any chance you can give it a try :pray: